### DISCUSSION TITLE
How can I set the DPI for the display in Skia/GTK?

### DISCUSSION THREAD

**Discussion by HakanL** [unoplatform/uno#14524] [Q&A] [answered]
I've found where it's reported (DisplayInformation), and how to override it in Linux Framebuffer. But I'm on Linux with Wayland/XWayland using GTK, is there a way to override the DPI for that platform? The issue is probably related to the fact that my HDMI touch screen doesn't provide EDID information, so the physical size comes back as 0x0mm. My 800x480 4.3" display should probably have a DPI around 220 if I calculated it correctly.

**Comment by jeromelaban** [comment, marked as answer]
There's no way to override the scaling at this time for GTK, it is following the scaling provided by the system.

**Reply by HakanL** [reply]
Ah, I was afraid of that, and probably explained why I couldn't find it :) Thanks though.
